apps/main/src/modules/trade/otc/useOtcExistentialDepositorMultiplier.ts

```diff
@@ -0,0 +1,15 @@
+import { queryOptions } from "@tanstack/react-query"
+
+import { TProviderContext } from "@/providers/rpcProvider"
+
+export const otcExistentialDepositorMultiplierQuery = (
+  rpc: TProviderContext,
+) => {
+  const { papi, isApiLoaded } = rpc
+
+  return queryOptions({
+    queryKey: ["trade", "otc", "existentialDepositorMultiplier"],
+    queryFn: () => papi.constants.OTC.ExistentialDepositMultiplier(),
+    enabled: isApiLoaded,
```

Comment:
I'd add staleTime: Infinite since it's a constant and move it to the constants in api folder